core/settings/base.py

```diff
@@ -144,5 +145,10 @@
 # Name of the URL where a user may login
 LOGIN_URL = 'auth_login'
 
-
 AUTH_USER_MODEL = 'register.User'
+
+OAUTH2_PROVIDER = {
+    'SCOPES': {'email': 'Viewing your email address.'},
+    'READ_SCOPE': ['email'],
```

Comment:
Isn't it possible to use these settings to filter the returned dict instead of doing it manually within the code? (see my previous comment)